pipelineSBOM.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Prerequisites 
+# apt install jq curl python3 python3-pip
+# WS_GENERATEPROJECTDETAILSJSON: true
+# WS_USERKEY
+# WS_APIKEY
+
+# Add the following after calling the unified agent in any pipeline file to create an SPDX output from the scanned project to the whitesource logs folder
+# then use your pipeline publish feature to save the whitesource log folder as an artifact as shown in the README
+#         curl -LJO https://raw.githubusercontent.com/whitesource-ft/ws-examples/main/pipelineSBOM.sh
+#         chmod +x ./pipelineSBOM.sh && ./pipelineSBOM.sh
+
+# More information & usage regarding WS SBOM generator can be found at https://github.com/whitesource-ps/ws-sbom-spdx-report
+
+WS_PROJECTTOKEN=$(jq -r '.projects | .[] | .projectToken' ./whitesource/scanProjectDetails.json)
+APIURL=https://saas.whitesourcesoftware.com
```

Comment:
@samq-ws , please add here (maybe as an argument)

Reply:
@tabac-ws It is unclear what you would like me to add....

Reply:
the APIURL endpoint is not used on the python execution.
This means all calls will be called to saas by default.
please add:
$APIURL=$1 (the URL will be a parameter on this bash file)